apps/crn-server/src/data-providers/research-outputs.data-provider.ts

```diff
@@ -288,6 +290,10 @@ export class ResearchOutputSquidexDataProvider
       labs: researchOutput.labs,
       teams: researchOutput.teams,
     });
+
+    if (updateOptions.publish) {
```

Comment:
Check out my comment in the controller tests - to me this line can stay as is and the only difference I would make is that I would tell the controller to always pass the `publish` option as true when the status is set to `published` - that way we would be calling the rest client `publish` method on every update - so the question is whether squidex might like it? If it doesn't - then I guess you could catch the error response although I would expect a 201